src/views/components/trace/trace-detail-chart-table.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+
+  <div class="trace-detail-chart-table">
+    <TraceContainer>
+      <Item v-for="(item, index) in tableData"  :data="item"  :key="'key'+ index" /> 
+    </TraceContainer>
+    <rk-sidebox :show.sync="showDetail" :title="$t('spanInfo')">
+      <div class="rk-trace-detail">
+        <h5 class="mb-15">{{$t('tags')}}.</h5>
+        <div class="mb-10 clear"><span class="g-sm-4 grey">{{$t('endpoint')}}:</span><span class="g-sm-8 wba">{{this.currentSpan.label}}</span></div>
+        <div class="mb-10 clear"><span class="g-sm-4 grey">{{$t('spanType')}}:</span><span class="g-sm-8 wba">{{this.currentSpan.type}}</span></div>
+        <div class="mb-10 clear"><span class="g-sm-4 grey">{{$t('component')}}:</span><span class="g-sm-8 wba">{{this.currentSpan.component}}</span></div>
+        <div class="mb-10 clear"><span class="g-sm-4 grey">Peer:</span><span class="g-sm-8 wba">{{this.currentSpan.peer||'No Peer'}}</span></div>
+        <div class="mb-10 clear"><span class="g-sm-4 grey">{{$t('error')}}:</span><span class="g-sm-8 wba">{{this.currentSpan.isError}}</span></div>
+        <div class="mb-10 clear" v-for="i in this.currentSpan.tags" :key="i.key"><span class="g-sm-4 grey">{{i.key}}:</span><span class="g-sm-8 wba">{{i.value}}</span></div>
+        <h5 class="mb-10" v-if="this.currentSpan.logs" v-show="this.currentSpan.logs.length">{{$t('logs')}}.</h5>
+        <div v-for="(i, index) in this.currentSpan.logs" :key="index">
+          <div class="mb-10 sm">
+            <span class="mr-10">{{$t('time')}}:</span>
+            <span class="grey">{{i.time | dateformat}}</span>
+          </div>
+          <div class="mb-15 clear" v-for="(_i, _index) in i.data" :key="_index">
+            <div class="mb-10">{{_i.key}}:</div>
+            <pre class="g-sm-8 mt-0 mb-0 sm" style="overflow:auto">{{_i.value}}</pre>
+          </div>
+        </div>
+      </div>
+    </rk-sidebox>
+  </div>
+</template>
+<script>
+import Item from './trace-chart-table/trace-item';
+import TraceContainer from './trace-chart-table/trace-container';
+/* eslint-disable */
+/* tslint:disable */
+export default {
+  components: {
+    Item,
+    TraceContainer
+  },
+  props: ['data'],
+  watch: {
+    'data'(val) {
+      this.tableData = this.formatData(this.changeTree())
+    }
+  },
+  data() {
+    return {
+      tableData: [],
+      diaplay: true,
+      selected: false,
+      segmentId:[],
+      showDetail: false,
+      list: [],
+      currentSpan: []      
+    }
+  },
+  methods: {
+    highlight () {
+      this.selected = true;
+    },
+    // 给增加层级关系
+    formatData (arr, level = 1, totalExec = null) {
```

Comment:
![image](https://user-images.githubusercontent.com/19775780/62281300-1a34eb00-b480-11e9-9015-794b1c24f60d.png)
![image](https://user-images.githubusercontent.com/19775780/62281320-2456e980-b480-11e9-9754-270028e89be3.png)

# very serious
The chart is incorrectly displayed.